docs/upgrade/UPGRADE-v8.0.0.md

```diff
@@ -69,6 +69,7 @@ There you can find links to upgrade notes for other versions too.
             - use Ivory\CKEditorBundle\Form\Type\CKEditorType;
             + use FOS\CKEditorBundle\Form\Type\CKEditorType;
             ```
+- update PHP version in your `composer.json` in `require` and `config` section to `7.2` because version `7.1` is no longer supported in Shopsys Framework ([#1066](https://github.com/shopsys/shopsys/pull/1066))
```

Comment:
The 7.1 is default version of PHP? I thought that people should use 7.3. If it is, I would recommend to change the upgrade note to "If you are using PHP 7.1 then,..."

Reply:
from the view of monorepo and project-base we want to fake version of php to minimal supported, because we use caret notation and we need to install dependencies of packages that supports the minimal supported version of php.